src/app/(marketing)/profile/_components/EditContributions.tsx

```diff
@@ -0,0 +1,61 @@
+"use client"
+import { Button } from "@/components/ui/Button"
+import {
+  Dialog,
+  DialogBody,
+  DialogCloseButton,
+  DialogContent,
+  DialogHeader,
+  DialogTitle,
+  DialogTrigger,
+} from "@/components/ui/Dialog"
+// import { profileSchema } from "@/lib/validations/profileSchema"
+import { Schemas } from "@guildxyz/types"
+// import { zodResolver } from "@hookform/resolvers/zod"
+import { Pencil } from "@phosphor-icons/react"
+import { FormProvider, useForm } from "react-hook-form"
+
+export const EditContributions = (
+  contribution: Schemas["ProfileContributionUpdate"]
+) => {
+  const form = useForm<Schemas["ProfileContributionUpdate"]>({
+    // resolver: zodResolver(profileSchema),
+    defaultValues: {
+      ...contribution,
+    },
+    mode: "onTouched",
+  })
+
+  async function onSubmit(values: Schemas["ProfileContributionUpdate"]) {
+    console.log("edit contributions submit", values)
+  }
+
+  return (
+    <Dialog>
+      <DialogTrigger asChild>
+        <Button variant="solid" size="icon">
```

Comment:
The button should be smaller & fully rounded (we can handle it here with a className & if we need to use the same button in multiple places, we could even add a variant to it)

Reply:
If I made it any smaller I think it would be too much. I will talk about this with the designers.